### PR TITLE
Docs: add note about management commands targeting a manager

### PR DESCRIPTION
**Pull request by thaJeztah**
Alternative for https://github.com/docker/cli/pull/1578

~~closes https://github.com/docker/cli/pull/1578~~ already merged

### CODE REVIEW

File: docs/reference/commandline/stack_deploy.md [hunk 1]
```diff
@@ -38,8 +38,11 @@ Options:
 
 ## Description
 
-Create and update a stack from a `compose` or a `dab` file on the swarm. This command
-has to be run targeting a manager node.
+Create and update a stack from a `compose` or a `dab` file on the swarm.
+
+> **Note**: This is a cluster management command, and must be executed on a swarm
```
Comment (thaJeztah) [inline docs/reference/commandline/stack_deploy.md]: Wasn't sure how to deal with k8s as an orchestrator here; this command can be run both on Swarm or Kubernetes, so we may need a different description to either make it generic enough to fit both use-cases, or extend the description to explicitly call out both use-cases

**Comment by thaJeztah** [comment]
This is a quick approach for this, but I think we can add annotations for this, so that;

- it can be used to automatically show/hide commands if the node is not a manager
- it can be included in the generated YAML docs, so that we can add this information automatically when generating the documentation; similar to the API-version and Orchestrator annotations;

screen shot 2018-12-23 at 12 33 00

**Comment by thaJeztah** [comment]
@ahh-docker @vdemeester

**Comment by ahh-docker** [comment]
LGTM

File: docs/reference/commandline/stack_deploy.md (hunk 1 above)
Comment (silvin-lubecki) [reply docs/reference/commandline/stack_deploy.md]: Maybe add a something like: ```**Note**:If you use swarm as an orchestrator, this is a cluster management command...``` ?

**Comment by SvenDowideit** [comment]
mmm, LGTM - though i'm saddened that I didn't build something to let you transclude the note from one place :D

**Comment by thaJeztah** [comment]
@silvin-lubecki slightly changed the wording for `docker stack` commands, to explicitly mention that a manager node must be targeted when using "swarm" as orchestrator.

**Pull request merged**